types/three/index.d.ts

```diff
@@ -1,6 +1,19 @@
 // Type definitions for three.js 0.84
 // Project: http://mrdoob.github.com/three.js/
-// Definitions by: Kon <http://phyzkit.net/>, Satoru Kimura <https://github.com/gyohk>, Florent Poujol <https://github.com/florentpoujol>, SereznoKot <https://github.com/SereznoKot>, HouChunlei <https://github.com/omni360>, Ivo <https://github.com/ivoisbelongtous>, David Asmuth <https://github.com/piranha771>, Brandon Roberge, Qinsi ZHU <https://github.com/qszhusightp>, Toshiya Nakakura <https://github.com/nakakura>, Poul Kjeldager Sørensen <https://github.com/s093294>, Stefan Profanter <https://github.com/Pro>, Edmund Fokschaner <https://github.com/efokschaner>
+// Definitions by: Kon <http://phyzkit.net/>
+//                 Satoru Kimura <https://github.com/gyohk>
+//                 Florent Poujol <https://github.com/florentpoujol>
+//                 SereznoKot <https://github.com/SereznoKot>
+//                 HouChunlei <https://github.com/omni360>
+//                 Ivo <https://github.com/ivoisbelongtous>
+//                 David Asmuth <https://github.com/piranha771>
+//                 Brandon Roberge
+//                 Qinsi ZHU <https://github.com/qszhusightp>
+//                 Toshiya Nakakura <https://github.com/nakakura>
+//                 Poul Kjeldager Sørensen <https://github.com/s093294>
+//                 Stefan Profanter <https://github.com/Pro>
+//                 Edmund Fokschaner <https://github.com/efokschaner>
+//                 John Rees <https://github.com/johnrees>
```

Comment:
Last time I tried this the DT parser actually couldn't handle it, but I think it might be fixed now. Please confirm, after this gets merged, that the package is published to npm with the right contributors, hopefully if that's correct then the PR mentions will work too.

Reply:
do you think it might've been related to Brandon Roberge not having a URL?

Reply:
After the PR is done I'll keep an eye on the result of the merge and see what happens anyway